lib/icon.js

```diff
@@ -40,12 +40,19 @@ const Chevron = ({ rotate, ...props }) => (
+  </SVGBase>
```

Comment:
`fill` should be a color variable here - `fill="var(--colors-primary)` would probably work? Other components do `fill="currentColor"`. 

This is necessary because the editor has dark mode, so #222 makes sense in light mode but will be invisible in dark mode. The `var(--colors-primary)` value will be #222 in light mode but the opposite in dark mode, as set by a theme provider component.

Reply:
great call out thanks!